src/client/ClientGroup.hpp

```diff
@@ -9,8 +9,10 @@
 class ClientGroup : public GameObject {
 
     public:
-        ClientGroup(sf::Vector2f position);
+        explicit ClientGroup(sf::Vector2f position);
```

Comment:
why was this change required?

Reply:
Codeclimate (cppcheck) complained about this.

Here is a good explanation about why this is good: https://stackoverflow.com/a/121163